contrib/docker/Dockerfile

```diff
@@ -34,19 +30,19 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
-# Build dcrd and other commands it provides
+# Build dcrd and other commands it provides.
 WORKDIR /go/src/github.com/decred/dcrd
 RUN git clone https://github.com/decred/dcrd . && \
     CGO_ENABLED=0 GOOS=linux \
     go install -trimpath -tags safe,netgo,timetzdata \
       -ldflags="-s -w" \
       . ./cmd/gencerts ./cmd/promptsecret
 
-# Build dcrctl
+# Build dcrctl.
 WORKDIR /go/src/github.com/decred/dcrctl
 RUN git clone https://github.com/decred/dcrctl . && \
     CGO_ENABLED=0 GOOS=linux \
-      go install -trimpath -tags safe,netgo -ldflags="-s -w"
+    go install -trimpath -tags safe,netgo -ldflags="-s -w"
 
 # Build entrypoint helper for the production image.
 WORKDIR /go/src/github.com/decred/dcrd/contrib/docker/entrypoint
```

Comment:
Why run `go mod init` here? Cant a `go.mod` and `go.sum` file for entrypoint be checked into git?

Reply:
I didn't want to check them in because it would make it a separate module in the repository that all of the tools that look for submodules would pick up and confuse them.

Reply:
I forgot to mention that also, I figured that if we ever want to start offering images on e.g. dockerhub via a separate repo, the code for `entrypoint` would need to live in that other repository and thus having a `go.mod` would result in the wrong path.  Creating it during the build is simpler and avoids those types of issues.